### ISSUE TITLE
Crash on adding an arrowed polyline to a LayerGroup that isn't displayed.

### ISSUE THREAD

**Issue by darrenfreeman** [slutske22/leaflet-arrowheads#15]
Error message:
`Cannot read property 'forEach' of undefined`

Steps to reproduce:
(Leaflet 1.7.1)

```
layerGroup = new L.LayerGroup();

marker = new L.Polyline(...).arrowheads(...);
marker.addTo(layerGroup);
```

Works fine if the layerGroup is currently added to the map, fails if it's currently not added, e.g. because the overlay is not selected in the layer chooser.

**Comment by slutske22** [comment]
Interesting...I'll take a look when I can.  Thanks for reporting this.

**Comment by darrenfreeman** [comment]
I already have a fix, just making you a PR.

**Comment by slutske22** [comment]
You rock. I'll take a look at that PR as soon as I can and test it out.  Once I see its working, I'll merge, publish to a new minor version, and close this issue.  Sit tight. Thanks!

**Comment by darrenfreeman** [comment]
There is another bug that I feel is related. Wong sized arrowheads seem to be created if the zoom level changes while the arrows are not displayed.

**Comment by slutske22** [comment]
I'd have to take a look.  If you could create a codesandbox displaying that issue, it would help alot to debug.  You have to be careful about *when* the arrowheads get created, vs when they actually get drawn.  If they're created, then the zoom changes, and *then* they're drawn, the `_update` function will not get called at the correct time, resulting in stale values for what the arrowheads should look like for a given zoom level.  Something to consider if you're going to look into it.  Otherwise, give me some time and I'll check it out.

**Comment by darrenfreeman** [comment]
Have created a new issue, Issue #17, as it has different steps to reproduce. I think this issue is fixed by the PR.

**Comment by darrenfreeman** [comment]
I believe #18 is the proper fix.

**Comment by slutske22** [comment]
Fixed by #18, thank you @darrenfreeman.  [Here is a CSB demonstrating the fix](https://codesandbox.io/s/leaflet-arrowheads-layergroup-bug-fixed-4wtl3), with leaflet-arrowheads up to 1.2.3.
